android/src/main/java/net/glxn/qrgen/android/QRCode.java

```diff
@@ -1,180 +1,273 @@
+
```

Comment:
same comment as for javase/QRCode. Please try to keep changes relevant. Reformatting and whitespace changes that change the whole file are unneccesary.

Reply:
Done!

Reply:
Awesome :)